.github/hooks/pre-commit

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+#
+# Install this using:
+# ln -s ../../.github/hooks/pre-commit .git/hooks/pre-commit
+#
+
+# Check if something changed in .circleci directory such that the config.yml needs to be updated
+if [[ $(git status --short | grep .circleci) ]]; then
```

Comment:
Think this does not really make sense. Remember the discussion we had on Integreat about this?
https://github.com/Integreat/integreat-app/pull/514/files

I would appreciate if, we copy this when the issue (IGAPP-855) is fixed or if you fix it now for lunes and integreat. We can also mark LUN-223-ticket as blocked by IGAPP-855.

Reply:
forget about this, too early in the morning. Everything fine.